### PR TITLE
PEP 458: Rename Role and Key Terms According to Issue #13

### PR DESCRIPTION
**Pull request by vladimir-v-diaz**
Addresses Issue #13.

* Update the figures and text to use the new terms.  `Online / offline` instances have been renamed to `on-pypi / off-pypi`, and `claimed / unclaimed` to `pypi-signed / developer-signed`.

* Expanded the section that discusses the `lazy bin walk` scheme.

### CODE REVIEW

File: pep-0458-tuf-online-keys.rst [hunk 1]
```diff
@@ -114,10 +114,10 @@ Threat Model
 
 The threat model assumes the following:
 
-* Offline keys are safe and securely stored.
+* Keys not kept on PyPI infrastructure are safe and securely stored.
 
-* Attackers can compromise at least one of PyPI's trusted keys stored online,
-  and may do so at once or over a period of time.
+* Attackers can compromise at least one of PyPI's trusted keys stored on PyPI
```
Comment (lvigdor) [inline pep-0458-tuf-online-keys.rst]: there should be a "the" after "stored on"

File: pep-0458-tuf-online-keys.rst [hunk 2]
```diff
@@ -505,34 +506,41 @@ Metadata Scalability
...
-download only a small number of TUF metadata files in order to update any
-project signed for by the *bins* role.  For example, applying this scheme to
-the previous repository resulted in pip downloading between 1.3KB and 111KB to
-install or upgrade a PyPI project via TUF.
+grow correspondingly.  For example, consider the *pypi-signed* role.  In August
+2013, it was found that the size of the *pypi-signed* metadata was about 42MB
+if the *pypi-signed* role itself signed for about 220K PyPI targets (which are
+simple indices and distributions).  This PEP does not delve into the details,
+but TUF features a so-called "`lazy bin walk`__" scheme that splits a large
+targets' metadata file into many small ones (bins).  Targets are then
```
Comment (lvigdor) [inline pep-0458-tuf-online-keys.rst]: 514: "targets' " should either be "targets" or "target's"

File: pep-0458-tuf-online-keys.rst [hunk 3]
```diff
@@ -553,18 +561,19 @@ recommends diversity of keys for certain applications.
...
-keys be independent of each other.  Different keys for online roles allow for
-each of the keys to be placed on separate servers if need be, and prevents side
-channel attacks that compromise one key from automatically compromising the
-rest of the keys.  Therefore, each of the *timestamp*, *snapshot*, and *bins*
-roles MUST require (1, 1) keys.
+The *timestamp*, *snapshot*, and *pypi-signed* roles require continuous
+delivery.  Even though their respective keys MUST be on-pypi, this PEP requires
+that the keys be independent of each other.  Different keys for pypi-signed
+roles allow for each of the keys to be placed on separate servers if need be,
+and prevents side channel attacks that compromise one key from automatically
```
Comment (lvigdor) [inline pep-0458-tuf-online-keys.rst]: 568: seems odd "compromise one key from automatically compromising the rest of the keys"
I think the first compromise should be replaced with "prevent"

File: pep-0458-tuf-online-keys.rst [hunk 4]
```diff
@@ -580,27 +589,27 @@ administrators or all PSF board members, and t > 1 (so that at least two
...
+attacks [21]_ unless diversity of keys is maintained.
+
+
+On-pypi and off-pypi Keys Recommended for Each Role
 -------------------------------------------------
 
-In order to support continuous delivery, the *timestamp*, *snapshot*, *bins*
-role keys MUST be online.
+In order to support continuous delivery, the *timestamp*, *snapshot*,
+*pypi-signed* role keys MUST be stored on PyPI infrastructure (on-pypi keys).
```
Comment (lvigdor) [inline pep-0458-tuf-online-keys.rst]: add "the" before PyPI

**Comment by vladimir-v-diaz** [comment]
@lvigdor 
Updates addressing your comments were made here: https://github.com/vladimir-v-diaz/interoperability-peps/commit/474c2f75d9c64b95d12f1c3f72de9d4bc7c1b509
Thanks for reviewing this PR!

**Pull request merged**